packages/backend/src/controllers/users/users.controller.ts

```diff
@@ -23,19 +30,25 @@ export class UsersController {
 
   @Post()
   @ApiOperation({
-    summary: 'create a new user resource',
+    summary: 'Create a new user resource',
     description: 'User must be linked to a firebase ID',
   })
-  @ApiCreatedResponse({
+  @ApiBadRequestResponse({
     description: 'User successfully created',
     type: UserResponseDto,
   })
+  @ApiBadRequestResponse({ description: 'no user name provided' })
   async create(@Body() createUserDto: CreateUserDto): Promise<UserResponseDto> {
     const userDoc = await this.userStoreService.create(createUserDto);
     const houseDoc = await this.houseStoreService.findOne(userDoc.house);
+    const userName = userDoc?.name && createUserDto.name;
```

Comment:
Just wondering if there is any functionality currently to pull the users name from the firebase token? That should be implemented for this check to make sense, unless I have misunderstood

Reply:
yeah if they sign in with a google account it will have a name attached

Reply:
emails don't have that hence why we have to have another way